helm/circles-graphprotocol/values.yaml

```diff
@@ -45,4 +45,15 @@ acmeServer: https://acme-v02.api.letsencrypt.org/directory
 # ethereum networks:
 network: xdai
 rpcNode: xdai-nethermind.xdai
-rpcPort: 8545
\ No newline at end of file
+rpcPort: 8545
+
+# CORS entried allowed: 
+CORS_allowed:
+  - "https://relayr.circles.garden"
```

Comment:
typo?

Reply:
Also this list should be reviewed

Reply:
yes, if you tell me which endpoint remove and add will change them

Reply:
fixed typo